source/docs/software/advanced-controls/introduction/tuning-vertical-arm.rst

```diff
@@ -54,66 +65,119 @@ Simulation - Vertical Arm
 
 .. raw:: html
 
-   </details> <br>
+    <div class="viz-div">
+      <div id="arm_pid_container">
+         <div class="col" id="arm_pid_plotVals"></div>
+         <div class="col" id="arm_pid_plotVolts"></div>
+      </div>
+      <div class="flex-grid">
+         <div class="col" id="arm_pid_viz"></div>
+         <div id="arm_pid_ctrls"></div>
+      </div>
+      <script>
+         arm_pidf = new VerticalArmPIDF("arm_pid");
+         arm_pidf.runSim();
+      </script>
+    </div>
+
+The core reason the feedback-only control strategy fails for the vertical arm is gravity.  The external force of gravity requires a constant :term:`control effort` to counteract even when at rest at the setpoint, but a feedback controller does not typically output any control effort when at rest at the setpoint (unless integral gain is used, which we can see clearly in the simulation is laggy and introduces oscillations).
+
+We saw in the feedforward-only example above that an accurate feedforward can track slow, smooth velocity setpoints quite well.  Combining a feedforward controller with the feedback controller gives the smooth velocity-following of a feedforward controller with the stable long-term error elimination of a feedback controller.
+
+Tuning the combined arm controller is simple - we first tune the feedforward controller following the same procedure as in the feedforward-only section, and then we tune the PID controller following the same procedure as in the feedback-only section.  Notice that PID portion of the controller is *much* easier to tune "on top of" an accurate feedforward.
+
+.. raw:: html
+
+   <details>
+     <summary>Tuning Solution</summary><br>
+
+<TODO: include kv, ka>
+
+In this particular example, reasonable values for the constants are :math:`K_g = 5.92465`, :math:`K_p = 6.0`, and :math:`K_d = 2.0`. These should produce good results at all setpoints.
+
+Once tuned properly, the combined controller should accurately track a smoothly moving setpoint, and also accurately converge to the setpoint over time after a "jump" command.
+
+A Note on Feedforward and Static Friction
+-----------------------------------------
 
+For the sake of simplicity, the simulations above omit the :math:`K_s` term from the WPILib SimpleMotorFeedforward equation.  On actual mechanisms, however, this can be important - especially if there's a lot of friction in the mechanism gearing.
 
-This shows how adding a carefully-chosen feedforward not only simplifies the calibration process, but produces better behavior at a wide range of setpoints.
+In the case of a vertical arm or elevator, :math:`K_s` can be somewhat tedious to estimate separately from :math:`K_g`.  If your arm or elevator has enough friction for :math:`K_s` to be important, it is recommended that you use the :doc:`WPILib system identification tool </docs/software/pathplanning/system-identification/introduction>` to determine your system gains.
```

Comment:
somewhat tedious - yup, that's an understatement. Thinking through about 4 experiments, I can't think of one that accurately separates the two.